src/assets/localization/languages/fi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "Azimuth": "Atsimuutti",
+    "Dawn": "Sarastus",
+    "Dusk": "Hämärä",
+    "Elevation": "Korkeus",
+    "Noon": "Keskipäivä",
+    "Sunrise": "Auringonnousu",
+    "Sunset": "Auringonlasku"
+  }
```

Comment:
There should probably be a newline at the end of file.

Reply:
Good catch! I added a newline to the end of file and fixed the indentation to match the other locale-files.